plugins/scaffolder-backend/src/service/router.ts

```diff
@@ -670,7 +624,9 @@ export async function createRouter(
         secrets: {
           ...body.secrets,
           ...(token && { backstageToken: token }),
+          initiatorCredentials: JSON.stringify(credentials),
```

Comment:
No need to pass through secrets here right? We're already passing the credentials in the top-level options